pkg/pgmodel/cache/inverted_labels_cache.go

```diff
@@ -0,0 +1,60 @@
+package cache
+
+import (
+	"fmt"
+
+	"github.com/timescale/promscale/pkg/clockcache"
+)
+
+type LabelInfo struct {
+	LabelID int32
+	Pos     int32
+}
+
+type LabelKey struct {
+	MetricName, Name, Value string
+}
+
+func NewLabelKey(metricName, name, value string) LabelKey {
+	return LabelKey{MetricName: metricName, Name: name, Value: value}
+}
+
+func (lk LabelKey) len() int {
+	return len(lk.MetricName) + len(lk.Name) + len(lk.Value)
+}
+
+func NewLabelInfo(lableID, pos int32) LabelInfo {
+	return LabelInfo{LabelID: lableID, Pos: pos}
+}
+
+func (li LabelInfo) len() int {
+	return 8
+}
+
+// Label key-pair -> (id,pos) cache
```

Comment:
What is this pos? We should add some comments explaining that id, and pos means. is id from the label_key table, or is it from label table, similarly for pos.

Reply:
Sure, will add some comments